spec/class.dd

```diff
@@ -1089,7 +1085,6 @@ assert(foo.x == int.init);  // object is still accessible
 
 $(GRAMMAR
 $(GNAME Deallocator):
-    $(D delete) $(GLINK2 function, Parameters) $(D ;)
     $(D delete) $(GLINK2 function, Parameters) $(GLINK2 function, FunctionBody)
```

Comment:
Hm, it seems that delete declarations are no longer accepted by the parser.

```d
class A
{
    delete(void *p) {}  
}
```

This fails with a parser error since  2.091.1.
Since class allocators/deallocators both end up with compilation errors, maybe it's just better if we
remove them altoghether from the spec?

Reply:
For this specific line deletion, please see above.

For the general topic of describing old language syntax, we can use `GDEPRECATED`.

There are some use cases to allow the grammar specify also previous versions of the language. If the grammar is used for anything other than develop a conforming implementation, such as code analysis tooling or (as is my case) syntax highlighting / editor support. Such tools have reasons to support previous versions of the language too.

We can use CSS or such to make removed / deprecated parts of the grammar clearly display as such (e.g. using 50% transparency or strikethrough).